db/flush_job_test.cc

```diff
@@ -114,9 +99,37 @@ class FlushJobTest : public testing::Test {
     ASSERT_OK(s);
   }
 
+  void SetUp() override {
+    EXPECT_OK(env_->CreateDirIfMissing(dbname_));
+
+    // TODO(icanadi) Remove this once we mock out VersionSet
+    NewDB();
+
+    db_options_.env = env_;
+    db_options_.fs = fs_;
+    db_options_.db_paths.emplace_back(dbname_,
+                                      std::numeric_limits<uint64_t>::max());
+    db_options_.statistics = ROCKSDB_NAMESPACE::CreateDBStatistics();
```

Comment:
Minor: `ROCKSDB_NAMESPACE` seems redundant here.

Reply:
Yes it does. I can remove it.